src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -673,5 +674,47 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
         }    
         
     }
-    
+
+    /**
+     * @description Detect if any opportunities exists where the same contact
+     * is associated with that opportunity multiple times through multiple
+     * opportunity contact role records.
+     */
+     public void verifyNoDuplicateContactRoles() {
+        try {
+            List<AggregateResult> results = [
+                SELECT
+                    COUNT(Id),
+                    OpportunityId,
+                    ContactId
+                FROM OpportunityContactRole
+                GROUP BY OpportunityId, ContactId
+                HAVING Count(Id) > 1
```

Comment:
@davidhabib, you're correct.  I will add more filters to this query (and the displayed query).  I was confusing this with a check against `IsPrimary` on OCR (which is not explicitly checked in the rollup code).